src/js/components/modals/InstallPackageModal.js

```diff
@@ -89,19 +92,29 @@ class InstallPackageModal
         currentTab: "defaultInstall",
         truncatedPreInstallNotes: true
       });
+
+      return;
     }
-    // If Opening
-    if (!props.open && nextProps.open) {
-      // If default - auto-install
-      if (!nextProps.advancedConfig) {
-        this.handleInstallPackage();
-      } else {
-        // Show advanced install
-        this.setState({
-          currentTab: "advancedInstall"
-        });
-      }
+
+    if (nextProps.isBetaPackage) {
```

Comment:
`#componentWillReceiveProps` will get called anytime props change. Should you remove the conditional, if what you say is true?